llama_toolchain/configs/distributions/generate_distribution_specs.py

```diff
@@ -0,0 +1,21 @@
+from llama_toolchain.core.distribution_registry import *
```

Comment:
This file seems odd. This seems to write some YAML files into distribution registry. But `available_distribution_specs` does the opposite -- namely just glob them all and construct Distribution specs out of them?